pyproject.toml

```diff
@@ -52,7 +52,6 @@ pandas = "^2.1.0"
 pgvector = "^0.2.3"
 psycopg2-binary = "^2.9.9"
 snowflake-connector-python = "^3.2"
-pytimeparse = "^1.1.8"
```

Comment:
Sorry, should've clarified in the other comment. `dateparser` is being used by a tool which means the dependencies are being loaded at runtime from the tool's `requirements.txt`. To use it in the framework we _also_ need to install it here.

Reply:
If the unit tests pass, it is because the tests for the `DateTime` tool happened to run first, installing the package.

Reply:
Gets me every time. Do you recommend a way to test that dependencies are correctly defined or is this muscle memory at this point?

Reply:
Muscle memory hahah. Maybe in the future we can run our tool tests after everything else.